src/core/ScreenSpace.tsx

```diff
@@ -0,0 +1,22 @@
+import * as React from 'react'
+import { Group } from 'three'
+import mergeRefs from 'react-merge-refs'
+import { useFrame } from '@react-three/fiber'
+
+export type ScreenSpaceProps = {
+  depth?: number
+} & JSX.IntrinsicElements['group']
+
+export const ScreenSpace = React.forwardRef<Group, ScreenSpaceProps>(({ children, depth = -1, ...rest }, ref) => {
+  const localRef = React.useRef<Group>(null!)
+
+  useFrame(({ camera }) => {
+    localRef.current.quaternion.copy(camera.quaternion)
+    localRef.current.position.copy(camera.position)
+  })
+  return (
+    <group ref={mergeRefs([ref, localRef])} {...rest}>
+      <group position-z={-depth}>{children}</group>
+    </group>
+  )
+})
```

Comment:
Do note that these camera transforms are in local space, so this can break if the camera is parented in some way. Alternatively, you can portal children into the camera and keep a single group for depth:

```js
import { createPortal, useThree } from '@react-three/fiber'

export const ScreenSpace = React.forwardRef<THREE.Group, ScreenSpaceProps>(function ScreenSpace(
  { children, depth = -1, ...rest },
  ref,
) {
  const camera = useThree((state) => state.camera)

  return (
    <>
      {createPortal(
        <group {...rest} ref={ref} position-z={-depth}>
          {children}
        </group>,
        camera,
      )}
    </>
  )
```

Reply:
Hi, i need to test both options, but can do that next week prolly 

The point of using:
```
localRef.current.quaternion.copy(camera.quaternion)
localRef.current.position.copy(camera.position)
```
Was that it supports the Html tag.
Also to take a note there is this approach witch is kind`a resonable:
https://discourse.threejs.org/t/how-to-build-a-hud-in-a-single-scene-with-a-single-camera/16108/2
```
camera.add(yourobject3d);
```